### PR TITLE
Document Service Usage

### PR DESCRIPTION
**Pull request by FrazerSeymour**
This PR adds documentation on how to set up and run a Superpeer on an Ubuntu server. It also updates the path in `superpeer.service` to be consistent with those instructions.

### CODE REVIEW

**Comment by FrazerSeymour** [comment]
[You can click here to read the rendered README file.](https://github.com/RightMesh/Superpeer/blob/feature/systemctl-docs/README.md)

File: README.md [hunk 1]
```diff
@@ -1,63 +1,79 @@
...
-in vagrant, you'll need to install vagrant and virtualbox. This will
-install the parity wallet within the vm so you don't accidentally
-clobber your local wallet.
+This minimalist RightMesh "Superpeer" implementation provides two functionalities: enabling the forwarding of data packets from one geographically separate mesh to another, and facilitating transactions involving RightMesh Tokens. The second functionality includes recording transactions to the Ethereum blockchain.
+
+## Running the Superpeer
+
+Superpeer is a Gradle application - it can be built with `gradlew build`, and a binary can be generated with `gradlew installDist`.
+
+If run without arguments Superpeer polls STDIN for input - typing `exit` will shut down RightMesh and stop the application. Unless you are developing/debugging, you will likely want to run Superpeer with the `-h | --headless` flag, which doesn't poll for input and responds to SIGINT signals (e.g. can be killed cleanly with `Ctrl+C` or task managers).
```
Comment (compscidr) [inline README.md:9]: awesome - this stuff is cool!

File: README.md [hunk 2]
```diff
@@ -1,63 +1,79 @@
...
+
+## Connecting to your Superpeer
+
+By default RightMesh devices with internet connections will connect to a Superpeer operated by RightMesh at `research.rightmesh.io`. In the future the goal is to have a network of Superpeers, with Superpeers implemented and operated by both RightMesh and community members.
+
+If you would like to have devices use your Superpeer instance for testing, you can hard-code its address into a RightMesh application to make it use your instance instead of the production network. You can do this by passing the address as a parameter to a special MeshManager constructor that is available in the developer version of the library. Note that this will never be an option in production - all RightMesh applications will use the same network, so that the owner of the device can specify their preferences for where/when/how to connect to Superpeers.
```
Comment (compscidr) [inline README.md]: We should probably be consistent with terminology - do we want to call it release or production? androids terminology for the target itself is release.

File: README.md [hunk 3]
```diff
@@ -1,63 +1,79 @@
...
 
-Todo: detect in gradle or use a set variable to specify whether we are
-running on a local machine or AWS instance. Also add an option to set
-a remote parity machine (if the superpeer wants to separate funciton).
+If you would like to send data to a Java application that makes use of RightMesh in a production application, you will need to find out its MeshID and send data to it through the RightMesh library.
```
Comment (compscidr) [inline README.md]: Not sure why this needs to be distinguished from an other RightMesh node - a java node and and android behave the same way in this respect.

File: README.md [hunk 4]
```diff
@@ -1,63 +1,79 @@
...
 
-Verify parity is running:
-`ps ax | grep parity`
+This repo contains everything needed to set up a Superpeer as a service on Ubuntu. These steps will help you get started:
 
-Compile against the latest library version
-`./gradlew build --refresh-dependencies`
+1. Set up an Amazon EC2 Instance running Ubuntu
+    - A Micro instance is fine, or any other Ubuntu server
+    - Make sure port 22/TCP is open for SSH, 40,000/UDP for the Superpeer, and both 30303/TCP and 30303/UDP for Parity to communicate with the Ethereum network.
```
Comment (compscidr) [inline README.md:31]: We may want to have the parity portion as an optional step. - it shouldn't be required to run parity if you don't want to.

File: README.md [hunk 5]
```diff
@@ -1,63 +1,79 @@
...
+2. SSH Into the server.
+    - `ssh ubuntu@your.ip.address.here`
+3. Install Java
```
Comment (compscidr) [inline README.md:34]: Java 8

File: README.md [hunk 6]
```diff
@@ -1,63 +1,79 @@
...
+    - `sudo add-apt-repository ppa:webupd8team/java`
+    - `sudo apt-get update`
+    - `sudo apt-get install oracle-java8-installer`
+4. Clone the Superpeer repository and navigate to the directory.
+    - `git clone https://github.com/rightmesh/superpeer`
+    - `cd superpeer`
+5. Update build.gradle with your credentials from the [RightMesh developer portal](https://developer.rightmesh.io/keys/).
+6. Install and configure Parity
```
Comment (compscidr) [inline README.md:42]: Should we mention explicitly the parity version here? Also sidenote - who makes sure we keep up with parity updates in this repo?

File: README.md [hunk 7]
```diff
@@ -1,63 +1,79 @@
...
-script. This will install parity from .deb (along with all dependencies)
-and start it up.
+# RightMesh Superpeer
 
-Otherwise you can uncomment the last few lines in the 
-build.gradle script to run parity in vagrant. If you wish to run it
-in vagrant, you'll need to install vagrant and virtualbox. This will
-install the parity wallet within the vm so you don't accidentally
-clobber your local wallet.
+This minimalist RightMesh "Superpeer" implementation provides two functionalities: enabling the forwarding of data packets from one geographically separate mesh to another, and facilitating transactions involving RightMesh Tokens. The second functionality includes recording transactions to the Ethereum blockchain.
```
Comment (compscidr) [inline README.md:3]: Should we mention these instructions assume Ubuntu? Also is there a specific Ubuntu version?

**Review by compscidr — APPROVED** [review]
Minor comments, looks good.

File: README.md (hunk 2 above)
Comment (FrazerSeymour) [reply README.md]: Now that I think about it I wonder if borrowing terminology from the crypto space would make more sense, something like the "RightMesh Mainnet".

"Release" is too ambiguous I think, like a release is a thing but a release can be outdated, it doesn't refer to the _latest_ release.

"Production" tends to imply the thing that is being put in front of customers but maybe doesn't get across what I mean here. I feel like beta releases of some software would still use the "Mainnet", even though they aren't "Production."

File: README.md (hunk 3 above)
Comment (FrazerSeymour) [reply README.md]: This was a holdover from Superpeer doubling as say servers, which might become a thing in the future with App Superpeers.

So it's here to explain "if you want a node to act like a centralized 'server' in a traditional sense and have it run on a server, use RightMesh not a Superpeer."

Comment (FrazerSeymour) [reply README.md]: Like that a MeshID is the way to talk to it, not it's IP address.

File: README.md (hunk 2 above)
Comment (raycho) [reply README.md]: I agree with using the term `Mainnet`, I think it makes more sense

**Pull request merged**